mllib/src/test/scala/org/apache/spark/ml/classification/NaiveBayesSuite.scala

```diff
@@ -46,6 +51,44 @@ class NaiveBayesSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(model.theta.map(math.exp) ~== thetaData.map(math.exp) absTol 0.05, "theta mismatch")
   }
 
+  def expectedMultinomialProbabilities(model: NaiveBayesModel, feature: Vector): Vector = {
```

Comment:
Can you modify the versions of this test code which are in the spark.mllib test suite so that you can reuse the code, rather than making a duplicate?  Same for expectedBernoulliProbabilities, validateProbabilities.